src/toxcore/friend_connection/packet/alive.rs

```diff
@@ -1,8 +1,8 @@
+
 /*! Alive struct
 */
 
 use toxcore::binary_io::*;
-
```

Comment:
No need to move the empty line to the beginning.

Reply:
done